core/core.ex

```diff
@@ -48,7 +51,14 @@ defmodule AntikytheraCore do
     System.put_env("ERL_LIBS", new_value)
   end
 
-  defp establish_connections_to_other_nodes(tries_remaining \\ 3) do
+  @connection_retry_interval_ms 5_000
+
+  defunpt calculate_connection_retry_count_from_health_check_grace_period() :: non_neg_integer do
```

Comment:
Ensure that the return value is `>= 1`. If it is 0, Antikythera doesn't connect to other nodes.

Reply:
Thanks, it was dangerous logic.
An a9a node should try to connect to other nodes at least 1 time.
Fixed in https://github.com/access-company/antikythera/pull/168/commits/746e621e2643650452f75bd628cfb0f93ea7df1d